pkg/build/node/import.go

```diff
@@ -50,8 +50,9 @@ func (c *containerdImporter) Prepare() error {
 
 func (c *containerdImporter) LoadCommand() exec.Cmd {
 	return c.containerCmder.Command(
-		"ctr", "--namespace=k8s.io", "images", "import", "--no-unpack", "-",
+		"ctr", "--namespace=k8s.io", "images", "import", "--all-platforms", "--no-unpack", "-",
 	)
+	// TODO: ideally we do not need this in the future. we have fixed at least one image
```

Comment:
normally I would place the TODO before the line that we'll remove, but it's not terribly important here